specs/common/format_amounts.rst

```diff
@@ -0,0 +1,39 @@
+=====================
+How to format amounts
+=====================
+
+This doc specifies the different types of formatting amounts inside the Safe apps.
+
+
+Full amount
+-----------
+
+Used when the entire amount matters. 
+
+- No truncation, always show all numbers.
+- Thousands and decimal separators are used according to user's locale.
+
+
+Short amount
+-------------------
+
+Used in cases where horizontal screen real estate is limited and the complete amount is not super important. 
+
+- Cut off after the 5th decimal, no matter how many decimals there are: 0.12345
+- Remove trailing zeroes, i.e. display 0.10000 as 0.1.
+- Use the 5 decimals up until 999.99999
+- Use 1 decimal less from 1,000.0001 until 9,999.9999 (Use comma as thousands-separator)
+- Use 1 decimal less from 10,000.001 until 99,999.999
+- Use 1 decimal less from 100,000.01 until 999,999.99
+- Use 1 decimal less from 1,000,000.1 until 9,999,999.9
```

Comment:
I think it is more common to use 2 decimals at least.

Reply:
For fiat currencies it is, yes, because there are not more decimals. For other numbers, why should 2 be more common?

Besides that, the bigger the amount before the decimal separator gets, the less important the numbers after it become. That's why we tried to find a solution that works for most numbers with the given space.

Please check the detailed discussions on this in https://github.com/gnosis/safe/issues/62

Everyone agree to that solution for now in a Safe team weekly, hence it's in this spec.

I'm happy to reconsider, but please provide more details on how your solution would look like with all kinds of amounts and the given space.